api/SpeechRecognition.json

```diff
@@ -309,7 +309,7 @@
               "notes": "You'll need to serve your code through a web server for recognition to work."
             },
             "webview_android": {
-              "version_added": true,
+              "version_added": false,
```

Comment:
Why is this (`continuous`) set to false when it’s supported on Chrome for Android?

Same question for other true &rarr; false changes below.

Reply:
Ah, I see what it is -- this interface is supported under a `webkit` prefix, so the collector isn't getting the correct results.  I'll close this PR for now.